### PR TITLE
Audit Item L08

### LINKED ISSUE #29: Resolve audit item L08

### PR DESCRIPTION
**Pull request by mds1**
Closes #29

### CODE REVIEW

File: contracts/Org.sol [hunk 1]
```diff
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: BSD 3-Clause
 
 pragma solidity ^0.6.10;
+pragma experimental ABIEncoderV2;
```
Comment (CantelopePeel) [inline contracts/Org.sol:4]: What is this for?

**Review by CantelopePeel — COMMENTED** [review]
Any reason why you are emitting the event before making the state change?

**Comment by mds1** [comment]
> Any reason why you are emitting the event before making the state change?

No reason really, if you prefer it afterwards we can move it.

File: contracts/Org.sol (hunk 1 above)
Comment (mds1) [reply contracts/Org.sol:4]: ABIEncoderV2 is an optional pragma that enables arbitrary encoding/decoding of structs. This is required for the events where we emit a struct, e.g. `emit GrantFinalized(grant);` and `emit ClaimCreated(newClaim);`. So without this pragma, you'd have to, for example, individually emit each field of the struct as a separate event parameter.

You can check out the docs on ABIEncoderV2 [here](https://solidity.readthedocs.io/en/v0.6.12/layout-of-source-files.html#experimental-pragma). Even though it's called an "experimental" pragma, it is no longer consider experimental (they just kept the pragma the same) and is safe to use.

**Comment by mds1** [comment]
Ah noticed there's some conflicts to resolve—I can resolve these tomorrow

**Comment by mds1** [comment]
Conflicts resolved, also ran prettier over the contracts since it seems some of the auto-formatting was lost in previous merges

**Comment by CantelopePeel** [comment]
Okay, I will double check this. I already have this merged locally and have added tests for the new events... Just need to push

**Comment by CantelopePeel** [comment]
Tests added, waiting for them to pass here then will merge.

File: contracts/Org.sol (hunk 1 above)
Comment (CantelopePeel) [reply contracts/Org.sol:4]: Sg

**Review by CantelopePeel — APPROVED** [review]
LGTM

**Pull request merged**